tests/yam/agama/boot_agama.pm

```diff
@@ -48,12 +49,18 @@ sub run {
     }
     my @params = split ' ', trim(get_var('EXTRABOOTPARAMS', ''));
 
-    $grub_menu->expect_is_shown();
-    $grub_menu->edit_current_entry();
-    $grub_entry_edition->move_cursor_to_end_of_kernel_line();
-    $grub_entry_edition->type(\@params);
-    $grub_entry_edition->boot();
-    $agama_up_an_running->expect_is_shown();
+
+    if (is_pvm_hmc()) {
```

Comment:
So we will only support hmc and not spvm for now? Just don't know the reason.

Reply:
spvm is the old console, with agama we should try the newest by default, but actually we could move in sles 15 in some point to hmc too, it is just was too messy our path in the squad with that backend.